src/components/player.js

```diff
@@ -99,8 +99,11 @@ export default class Player extends BaseObject {
    * define a poster by adding its address `poster: 'http://url/img.png'`. It will appear after video embed, disappear on play and go back when user stops the video.
    * @param {String} [options.playbackNotSupportedMessage]
    * define a custom message to be displayed when a playback is not supported.
+   * @param {Object} readyCallback (optional)
+   * If provided this will be called when the player has finished loading (just before the READY event is fired).
+   * It will be provided the player instance as the first argument.
    */
-  constructor(options) {
+  constructor(options, readyCallback) {
```

Comment:
Just discovered an issue with events which is that they can be fired before the `new Clappr.Player()` completes, meaning the user doesn't get chance to listen to them!

@leandromoreira's solution would allow people to listen to events before the creation of the player completes, if the listeners for them were registered internally before anything would fire them.